### DISCUSSION TITLE
[Feature] Automaticall albums based on Exif-data

### DISCUSSION THREAD

**Discussion by Zev-se** [immich-app/immich#11993] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Create album based on exif-data automatically. Say I want one album with all pictures from my iPhone 14. But also an album with all pictures taken in Greece. Or all photos taken between two dates. Or all photos taken with a specific lens. With all the data that is given this seems like a good way of automating an otherwise time consuming activity. I know someone on Reddit said they import photos from different cameras in different sub-folders to solve one of the above issues but there must be a better and more dynamic way.

If you combine this with the ability to create [custom exif-tags](https://github.com/immich-app/immich/issues/1965) this would become a very powerful feature.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by C-Otto** [comment]
There needs to be some manual action, otherwise you'd drown in albums you're not interested in (for example, I don't care about lenses and phone models). Given that, why don't you create albums based on the results you get after searching with a specific filter?

**Reply by Zev-se** [reply]
Oh yes, sorry. The idea is to be able to create an album (manually) and say this album contains all pictures that have this exif-data. It should from there be automatic. Good catch!

**Reply by C-Otto** [reply]
So you create the album first (manually) and then Immich automatically adds new assets to that album?

**Reply by Zev-se** [reply]
Yes

**Comment by C-Otto** [comment]
Related to #8331, possible duplicate of #1673

**Reply by Zev-se** [reply]
#1673 Seems to be mainly the same idea. Even if it extends on this with metadata Immich itself creates (face recognition).

**Reply by C-Otto** [reply]
Feel free to close this then :)
